src/EnergyPlus/StandardRatings.cc

```diff
@@ -362,12 +367,22 @@ namespace StandardRatings {
                 auto const SELECT_CASE_var(ChillerType);
 
                 if (SELECT_CASE_var == TypeOf_Chiller_ElectricEIR) {
+                    if (RedCapNum == 1.0) {
+                        // Get curve modifier values at rated conditions (load = 100%)
+                        ChillerCapFT_rated = CurveValue(state, CapFTempCurveIndex, EvapOutletTemp, CondenserInletTemp);
+                        ChillerEIRFT_rated = CurveValue(state, EIRFTempCurveIndex, EvapOutletTemp, CondenserInletTemp);
+
+                        // Report rated capacity and chiller COP
+                        PreDefTableEntry(state, state.dataOutRptPredefined->pdchMechRatCap, ChillerName, RefCap * ChillerCapFT_rated);
+                        PreDefTableEntry(state, state.dataOutRptPredefined->pdchMechRatEff, ChillerName, RefCOP / ChillerEIRFT_rated);
+                    }
+
```

Comment:
Retrieve rated chiller capacity and EIR at rated conditions and reports the rated chiller capacity and COP.

Reply:
If the user specified capacity, which is called the reference or nominal capacity, is not at the rated condition, then the *CAPFT_rated* modifier curve value is required to bring the user specified capacity (Cap_ref) to the rated condition. Similarly, the reference COP also needs to be corrected to the rated condition. I agree with these changes.